docs/content/api_core/api_getting_started_developers.rst

```diff
@@ -0,0 +1,226 @@
+
+Setting up your environment
+---------------------------
+
+So that you can access SimPEG from anywhere on your computer, you need to add
+it to your path. This can be done using symlinks. In your :code:`git` directory,
+create a directory called :code:`python_symlinks`.
+
+.. image:: ../../images/gitfolders.png
+    :align: center
+    :width: 40%
+
+Open a terminal in this directory and create a symlink for SimPEG ::
+
+    ln -s ../SimPEG/SimPEG .
+
+Then, in your shell, you need to add a :code:`PYTHONPATH` variable. For Mac and
+Linux, if you are using Z shell (`Oh My Zsh <http://ohmyz.sh/>`_ is used by a
+lot of SimPEG developers) or bash open the config in a text editor, ie::
+
+    nano ~/.zshrc
+
+or::
+
+    nano ~/.bash_profile
+
+and add a :code:`PYTHONPATH` variable::
+
+    export PYTHONPATH="$PYTHONPATH:/Users/USER/git/python_symlinks"
+
+and save and close. If you then restart the terminal, and run::
+
+    echo $PYTHONPATH
+
+the output should be::
+
+    /Users/USER/git/python_symlinks
+
+
+.. _getting_started_text_editors:
+
+Text Editors
+------------
+
+Sublime_ is a text editor used by many SimPEG developers.
+
+.. _Sublime: https://www.sublimetext.com/
+
+You can configure the Sublime so that you can use the sublime
+build (Tools / Build) to run Python_ code.
+
+Open your user settings
+
+.. image:: ../../images/sublimeSettings.png
+    :align: center
+    :width: 400
+
+and edit them to include the path to your :code:`python_symlinks`::
+
+    {
+    "added_words":
+    [
+        "electromagnetics"
+    ],
+    "ensure_newline_at_eof_on_save": true,
+    "extra_paths":
+    [
+        "/Users/USER/git/python_symlinks/"
+    ],
+    "font_size": 11,
+    "ignored_packages":
+    [
+        "Vintage"
+    ],
+    "translate_tabs_to_spaces": true,
+    "trim_trailing_white_space_on_save": true,
+    "word_wrap": false
+    }
+
+There are a few other things configured here. In particular you will want to
+ensure that :code:`"translate_tabs_to_spaces": true` is configured (Python_ is
+sensitive to tabs and spaces), that
+:code:`"trim_trailing_white_space_on_save": true` so that your git flow does
+not get cluttered with extra spaces that are not actually changes to code and
+that :code:`"ensure_newline_at_eof_on_save": true`, so that there is a blank
+line at the end of all saved documents. The rest are up to you.
+
+.. _getting_started_jupyter_notebook:
+
+Jupyter Notebook
+----------------
+
+.. image:: http://blog.jupyter.org/content/images/2015/02/jupyter-sq-text.png
+    :align: right
+    :width: 100
+
+The SimPEG team loves the `Jupyter notebook`_. It is an interactive
+development environment. It is installed it you used Anaconda_ and can be
+launched from a terminal using::
+
+    jupyter notebook
+
+
+.. _getting_started_if_all_is_well:
+
+If all is well ...
+------------------
+
+You should be able to open a terminal within SimPEG/examples and run an example, ie.::
+
+    python 01-basic/plot_inversion_linear.py
```

Comment:
![image](https://cloud.githubusercontent.com/assets/6361812/25667021/0a920a96-2fd7-11e7-9d32-0787ea51643c.png)
(we added more data, so I think it should be fine)